src/loglevels.cpp

```diff
@@ -42,49 +42,76 @@ namespace g3 {
          setLogLevel(level, true);
       }
 
+      void reset() {
+         g3::internal::g_log_levels = g3::internal::g_log_level_defaults;
+      }
+} // only_change_at_initialization
+
+
+namespace log_levels {
 
-      void setHighestLogLevel(LEVELS enabledFrom) {
+      void setHighest(LEVELS enabledFrom) {
```

Comment:
The description says
```
      /// Enable log level >= log_level. 
      /// log levels below will be disabled
      /// log levels equal or higher will be enabled.
```
But currently this only disables the one below it. 
*options*
1. change the description
2. push that all above the level should be enabled